polkadot/node/network/statement-distribution/src/v2/tests/mod.rs

```diff
@@ -174,7 +177,13 @@ impl TestState {
 			random_seed: [0u8; 32],
 		};
 
-		TestState { config, local, validators, session_info, req_sender }
+		let mut node_features = NodeFeatures::new();
+		if config.allow_v2_descriptors {
+			node_features.resize(4, false);
```

Comment:
we shouldn't hardcode 4 here

Reply:
It's just for test, I think this is fine.

Reply:
used `FirstUnassigned`